packages/ui/cypress/e2e/case-list/filter-cases/triggersExcludedAtUserLevel.cy.ts

```diff
@@ -0,0 +1,62 @@
+describe("Triggers excluded at user level", () => {
+    it("should not show excluded triggers in trigger groups and reason codes in the reason codes input when we click on Bails", () => {
+      cy.loginAs("SupervisorWithExcludedTriggers")
+      cy.visit("/bichard")
+
+      cy.get("input#bails").should("exist")
+      cy.get("input#bails").click()
+
+      cy.get(".govuk-checkboxes__item").contains("PR08 - Breach of bail").should("not.exist")
+      cy.get(".govuk-checkboxes__item").contains("PR10 - Conditional bail").should("exist")
+      cy.get(".govuk-checkboxes__item").contains("PR20 - Breach").should("exist")
+      cy.get(".govuk-checkboxes__item").contains("PR30 - Pre-charge bail application").should("exist")
```

Comment:
You can be more explicit with the finders:

```typescript
cy.get(".govuk-checkboxes__item #trpr0008 + label").should("not.exist")
cy.get(".govuk-checkboxes__item #trpr0010 + label").contains("PR10 - Conditional bail").should("exist")
cy.get(".govuk-checkboxes__item #trpr0020 + label").contains("PR20 - Breach").should("exist")
cy.get(".govuk-checkboxes__item #trpr0030 + label").contains("PR30 - Pre-charge bail application").should("exist")
```

(Remove the SonarLint duplication warning)

Reply:
Thanks for suggesting, applied changes to remove sonar warnings